recipes/pyspark/pysparkretailapp/evaluator.py

```diff
@@ -16,28 +16,16 @@
 #####################################################################
 
 from pyspark.sql.functions import col, avg, sqrt, abs, pow, lit
```

Comment:
Not all of these imports are being used.

Reply:
Unused imports have been removed.